index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>HTML Quiz | Chase Ramsey</title>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/stylez.css">
+	<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,400italic,300italic' rel='stylesheet' type='text/css'>
+</head>
+<body>
+
+<header id="main-header" class="clearfix">
+<h1><a href="#"><span class="u_logo">YOU</span>nicorn</a></h1>
+	<nav>
+		<ul>
+			<li><a href="#" alt="Contact">Contact</a></li>
```

Comment:
`alt` attributes are not allowed on anchor `a` tags. Only: `applet`, `area`, `img`, & `input` tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes